mythril/laser/smt/bitvec.py

```diff
@@ -139,44 +139,48 @@ def __gt__(self, other) -> Bool:
         union = self.annotations + other.annotations
         return Bool(self.raw > other.raw, annotations=union)
 
-    def __eq__(self, other) -> Bool:
+    #MYPY: fix complains about overriding __eq__
+    def __eq__(self, other: Union[int, "BitVec"]) -> Bool:  # type: ignore
         """Create an equality expression.
 
         :param other:
         :return:
         """
         if not isinstance(other, BitVec):
-            return Bool(self.raw == other, annotations=self.annotations)
+            return Bool(cast(z3.BoolRef, self.raw == other), annotations=self.annotations)
 
         union = self.annotations + other.annotations
-        return Bool(self.raw == other.raw, annotations=union)
+        # MYPY: fix complaints due to z3 overriding __eq__
+        return Bool(cast(z3.BoolRef, self.raw == other.raw), annotations=union)
 
-    def __ne__(self, other) -> Bool:
+    #MYPY: fix complains about overriding __ne__
```

Comment:
There should be a space after #, according to PEP8 standards

Reply:
Fixed in next commit